docs/reference/remap/replace.cue

```diff
@@ -0,0 +1,73 @@
+package metadata
+
+remap: functions: replace: {
+	arguments: [
```

Comment:
A new `count` argument has been added to this function (see https://github.com/timberio/vector/pull/4861)